specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2018-12-20/bms.json

```diff
@@ -3932,6 +3936,14 @@
         "isRehydrate": {
           "description": "Flag to identify that deferred deleted DS is to be moved into Pause state",
           "type": "boolean"
+        },
+        "policyName": {
+          "description": "Name of the policy used for protection.",
+          "type": "string"
+        },
+        "isArchiveEnabled": {
```

Comment:
>isArchiveEnabled [](http://example.com/codeflow?start=9&length=16)

String enums are preferred to booleans because they are easier for customers to use, allow additional values to be added, and can be designed to avoid invalid combinations with other switches. For example, this property could be archiveMethod: { None, Full } or similar.